pkg/comp-functions/functions/common/interfaces.go

```diff
@@ -18,6 +18,8 @@ type InfoGetter interface {
 	GetSecurity() *vshnv1.Security
 	InstanceNamespaceInfo
 	GetPDBLabels() map[string]string
+	GetServiceKubeType() vshnv1.PodTemplateLabelsManager
+	GetServiceKubeTypeName() string
 }
```

Comment:
I would change the name of those functions as "Service" is misleading here

Reply:
Can you suggest a better name?

Reply:
I think I found a solution :)